osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/firmware/UpdateFirmwareCommandExecutor.java

```diff
@@ -8,15 +8,13 @@
  */
```

Comment:
There is no unit test for this command executor.

Reply:
Correct there never was, it is purely test in Integration testing.

Reply:
I added some really cool testing stuff for this class